src/UNL/Peoplefinder/Driver/LDAP/StandardFilter.php

```diff
@@ -78,22 +82,10 @@ function __construct($inquery, $operator = '&', $wild = false)
                 $filter .= ")";
             }
             $filter .= ")";
-
-            //determine if a wildcard should be used
-            if ($wild) {
```

Comment:
Removing this obsoletes the `$wild` parameter. Was this intentional? It seems like this is an important part of partial searching.